### DISCUSSION TITLE
Click tracking webhook event

### DISCUSSION THREAD

**Discussion by JasonDevTech** [postalserver/postal#1804] [Feature suggestions]
Since setting up Postal tracking, I've noticed in the UI we can see if an email was opened or not, and the same for links. However, setting up a webhook, I've noticed that only link tracking sends a webhook request and not for email views. Would it be possible to also create another webhook event with the same payload as a link click but for email views instead? I believe this would allow us to get a better overview of user engagement rather than _having_ to click a link or view the UI every time.

**Comment by willpower232** [comment]
I think you are looking for the `MessageLoaded` event

**Reply by JasonDevTech** [reply]
I don't believe so, because I tested that already and the webhook never fires though the UI updates correctly.

**Reply by willpower232** [reply]
We're definitely receiving these events so I'm not sure why you wouldn't be. Does it behave differently if you tell the webhook to send you all events? Thats the only difference I can think off of the top of my head

![image](https://user-images.githubusercontent.com/1619102/149191528-8c65db67-ffdc-41a8-8910-7d9480b0752e.png)

**Reply by JasonDevTech** [reply]
I retried it again and it does work now, it was just extremely delayed in sending the request.

**Reply by willpower232** [reply]
Good to know, Postal doesn't come with that many worker processes by default. You might want to look at scaling up the worker to get a faster flow going
